plugin/modules.js

```diff
@@ -116,7 +116,7 @@
       var path = parentFile ? resolvePath(dirName(parentFile), word) : baseName(word)
       for (var prop in this.modules) {
         if (prop != parentFile && filter(path, prop, query)) {
-          if (/\.js$/.test(prop)) prop = prop.slice(0, prop.length - 3)
+          if (/\.jsx?$/.test(prop)) prop = prop.slice(0, prop.length - 3)
```

Comment:
Preferably, just store the regexp match and use `match[0].length` to get its actual length